src/main/java/Dul.java

```diff
@@ -0,0 +1,181 @@
+import java.util.Scanner;
```

Comment:
Coding standards followed. Clear naming for different functions. Might be good to add comments before each function to increase code understanding, but overall well-written.